docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `policyIdentifier` |              | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **2+**                 | SHOULD       | The CA SHOULD include additional Reserved Certificate Policy Identifiers for each type of Subscriber certificate that may be issued beneath it. |
+| \ \ \ \ `policyIdentifier` | MUST         | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3+**                 | MAY          | The CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **4**                  | MUST NOT     | The CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
 
-   For all other Subordinate CA Certificates, including Technically Constrained Subordinate CA Certificates:
+If the `policyQualifiers` is permitted and present within a `PolicyInformation` field, it MUST be formatted as follows:
 
-   This extension MUST be present and SHOULD NOT be marked critical[^**].
+| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| ---                                  | -            | -              | -----         |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP or HTTPS URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
+| Any other qualifier                  | MUST NOT     | -              | -             |
+
+##### 7.1.2.8.6 Extended Key Usage - Non-TLS CA
+
+The Issuing CA MUST verify that the Subordinate CA Certificate is authorized to issue certificates for each included extended key usage purpose. Multiple, independent key purposes (e.g. `id-kp-timeStamping` and `id-kp-codeSigning`) SHOULD NOT be present.
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST NOT     |
+| `id-kp-clientAuth`      | 1.3.6.1.5.5.7.3.2 | MAY          |
+| `id-kp-codeSigning`     | 1.3.6.1.5.5.7.3.3 | MAY          |
+| `id-kp-emailProtection` | 1.3.6.1.5.5.7.3.4 | MAY          |
+| `id-kp-timeStamping`    | 1.3.6.1.5.5.7.3.8 | MAY          |
+| `id-kp-OCSPSigning`     | 1.3.6.1.5.5.7.3.9 | MAY          |
```

Comment:
Is there a situation when id-kp-OCSPSigning could be included into a CA certificate?  I know of a billion reasons why we don't want to permit that.

Reply:
Good spot! This is a bug :)

(It's a bug in our existing specs, but I should have fixed this, since this is important to prohibit)

Reply:
Fixed! This should now appropriately clarify MUST NOT for OCSPSigning, and MAY for any other value :)